back/app/models/custom_form_logic.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class CustomFormLogic < ApplicationRecord
+  acts_as_list column: :ordering, top_of_list: 0, scope: %i[source_field_id target_field_id]
+
+  belongs_to :source_field, class_name: 'CustomField'
+  belongs_to :target_field, class_name: 'CustomField'
+
+  scope :ordered_on_target, lambda {
+    joins(:source_field).order('custom_fields.ordering', 'custom_form_logics.ordering')
+  }
+  scope :ordered_on_source, lambda {
+    joins(:target_field).order('custom_fields.ordering', 'custom_form_logics.ordering')
+  }
+
+  # TODO: validations
+  # - source and target fields present
+  # - source and target fields same custom form
+  # - source field cannot be page, target field must be page
+  # - source field before target field
+  # - condition_operator present and inclusion from list
+  # - one value attribute present, others nil
+  # - action present and inclusion from list
+
+  def satisfies?(custom_field_values)
```

Comment:
This method is not used.
Why would there be a link between this model and custom field values?